rest_sensors.cpp

```diff
@@ -1432,17 +1446,31 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                         }
                         if (data.length() > 0)
                         {
-                            if (sendTuyaRequest(task, TaskThermostat , DP_TYPE_ENUM, 0x6a, data))
+                            if (R_GetProductId(sensor) == QLatin1String("Tuya_THD MOES TRV"))
                             {
-                                updated = true;
+                                if (modeSet == "off")
+                                {
+                                    sendTuyaRequest(task, TaskThermostat , DP_TYPE_ENUM, 0x02, data);
```

Comment:
Didn't we have constants for the values like 0x02?

Reply:
done